index.js

```diff
@@ -68,28 +117,15 @@ class ExecutorRouter extends Executor {
      * @return {Promise}
      */
     _start(config) {
-        const annotations = config.annotations || {};
-        const executorType = annotations[ANNOTATION_EXECUTOR_TYPE];
-        const executor = this[executorType] || this.defaultExecutor; // Route to executor (based on annotations) or use default executor
-
-        return executor.start(config);
-    }
-
-    /**
-     * Stop a running or finished build
-     * @method _stop
-     * @param {Object} config               Configuration
-     * @param {Object} [config.annotations] Optional key/value object
-     * @param {String} config.buildId       Unique ID for a build
-     * @return {Promise}
-     */
-    _stop(config) {
-        const annotations = config.annotations || {};
-        const executorType = annotations[ANNOTATION_EXECUTOR_TYPE];
-        const executor = this[executorType] || this.defaultExecutor; // Route to executor (based on annotations) or use default executor
-
-        return executor.stop(config);
+        return this.verifyConnection()
+            .then(() => {
+                const options = this.getFullConfig(config);
+
+                // TODO: mark job as queued
```

Comment:
My understanding was that for the first pass we were only worried about `start`, not `status`. I can definitely add that in though.